vcfa/provider.go

```diff
@@ -88,6 +89,7 @@ var globalResourceMap = map[string]*schema.Resource{
 	"vcfa_certificate":                     resourceVcfaCertificate(),                 // 1.0
 	"vcfa_org_local_user":                  resourceVcfaLocalUser(),                   // 1.0
 	"vcfa_org_ldap":                        resourceVcfaOrgLdap(),                     // 1.0
+	"vcfa_ldap":                            resourceVcfaLdap(),                        // 1.0
```

Comment:
One thought - this is `provider` only resource. Should we explicitly name it `vcfa_provider_ldap`?
cc @lvirbalas

Reply:
Changed. Test still passes